src/SprykerSdk/Zed/Benchmark/Business/Helper/DataTable/DataTableHelper.php

```diff
@@ -0,0 +1,175 @@
+        );
+    }
+
+    /**
+     * @param array $tableConfigData
+     * @param \Generated\Shared\Transfer\GuiTableConfigurationTransfer $tableConfigTransfer
+     *
+     * @return \Generated\Shared\Transfer\GuiTableConfigurationTransfer
+     */
+    protected function mapTableConfigDataToGuiTableConfigurationTransfer(
+        array $tableConfigData,
+        GuiTableConfigurationTransfer $tableConfigTransfer
+    ): GuiTableConfigurationTransfer {
+        $tableConfigTransfer->fromArray($tableConfigData, true);
+
+        foreach ($tableConfigData as $key => $value) {
+            if (!array_key_exists('enabled', $value)) {
+                continue;
+            }
+            $nestedTransfer = $tableConfigTransfer->offsetGet($key);
+            if (method_exists($nestedTransfer, 'setIsEnabled')) {
+                $nestedTransfer->setIsEnabled($value['enabled']);
+            }
+        }
+
+        return $tableConfigTransfer;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\GuiTableConfigurationTransfer $tableConfigTransfer
+     * @param string $type
+     *
+     * @throws \SprykerSdk\Zed\Benchmark\Business\Exception\InvalidGuiTableDataSourceTypeException
+     *
+     * @return string
+     */
+    public function getUrlFromDataSource(GuiTableConfigurationTransfer $tableConfigTransfer, string $type): string
+    {
+        $dataSource = $tableConfigTransfer->getDataSource();
+        if (!$dataSource->getType() === $type) {
+            throw new InvalidGuiTableDataSourceTypeException(
+                sprintf('Unknown dataSource type "%s".', $type)
+            );
+        }
+
+        return $dataSource->getUrl();
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\GuiTableConfigurationTransfer $tableConfigTransfer
+     *
+     * @throws \SprykerSdk\Zed\Benchmark\Business\Exception\InvalidGuiTableFilterTypeException
+     *
+     * @return array
+     */
+    public function getAllFiltersWithRandomValues(GuiTableConfigurationTransfer $tableConfigTransfer): array
+    {
+        $filtersConfigTransfer = $tableConfigTransfer->getFilters();
+        if (!$filtersConfigTransfer->getIsEnabled() === true) {
+            return [];
+        }
+
+        $filtersWithValues = [];
+        foreach ($filtersConfigTransfer->getItems() as $filterTransfer) {
+            $randomValue = $this->createRandomValueForFilter($filterTransfer);
+            if ($randomValue !== null) {
+                $filtersWithValues[$filterTransfer->getId()] = $this->createRandomValueForFilter($filterTransfer);
+            }
+        }
+
+        return $filtersWithValues;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\GuiTableFilterTransfer $filterTransfer
+     *
+     * @throws \SprykerSdk\Zed\Benchmark\Business\Exception\InvalidGuiTableFilterTypeException
+     *
+     * @return array|string
+     */
+    protected function createRandomValueForFilter(GuiTableFilterTransfer $filterTransfer)
```

Comment:
This will lead to random results of the benchmark. Based on the value we pick, we might perform the request faster or slower. 

Let's just pick the first possible value? So the filter set is predictable.